backend/minors/REL.yaml

```diff
@@ -0,0 +1,188 @@
+excluded_major: 
+- REL
+contacts:
+- type: Director of Undergraduate Studies
+  name: Bryan Lowe
+  email: bdlowe@princeton.edu
+req_list:
+- name: REL Minor Courses
+  max_counted: 1
+  min_needed: 3
+  explanation: |-
+    Students must successfully complete five courses related to religion, fulfilling certain conditions.
+  double_counting_allowed: false
+  req_list:
+  - name: REL Courses
+    max_counted: 1
+    min_needed: 5
+    explanation: |-
+      At least four of these courses must be offered by the Department of Religion. One course may be 
+      offered elsewhere in the University, as long as it is approved as suitably relevant by the DUS. 
+      Courses cross-listed with REL may be considered pre-approved.
+    double_counting_allowed: false
+    pdfs_allowed: 1
+    course_list:
+    - REL ***
+  - name: Multiple areas of study
+    max_counted: 1
+    min_needed: 2
+    explanation: |-
+      The five courses should not all be focused on a single religious tradition or approach. 
+      That is, the five courses must include courses from at least two of the five “Areas of Study” in which 
+      the Department of Religion offers courses. The five Areas of Study are Ancient Judaism, Christianity, 
+      and Greek and Roman Religions; Islam; Religion in the Americas; Religions of Asia; and Philosophical and 
+      Ethical Approaches to Religion.
+    req_list:
+    - name: Religions of Asia
+      max_counted: 1
+      min_needed: 1
+      course_list:
+        - REL 225
+        - REL 226
+        - REL 228
+        - REL 229
+        - REL 322
+        - REL 326
+    - name: Ancient Judaism, Christianity, and Greek and Roman Religions
+      max_counted: 1
+      min_needed: 1
+      course_list:
+        - REL 230
+        - REL 251
+        - REL 252
+        - REL 350
+        - REL 352
+        - REL 353
+    - name: Islam
+      max_counted: 1
+      min_needed: 1
+      course_list:
+        - REL 235
+        - REL 236
+        - REL 240
+        - REL 328
+        - REL 334
+        - REL 335
+        - REL 336
+        - REL 338
+    - name: Religion in America
+      max_counted: 1
+      min_needed: 1
+      course_list:
+        - REL 258
+        - REL 319
+        - REL 357
+        - REL 358
+        - REL 360
+        - REL 367
+    - name: Philosophical and Ethical Approaches to Religion
+      max_counted: 1
+      min_needed: 1
+      course_list:
+        - REL 242
+        - REL 261
+        - REL 311
+        - REL 312
+        - REL 317
+        - REL 346
+        - REL 347
+        - REL 363
+        - REL 364
+  - name: Focus on an area of study
+    max_counted: 1
```

Comment:
We should reorganize the reqs so we only have the "Multiple Areas of Study", and make it have min_needed 2. Then make each of the area a subreq under that with min_needed = 1 and max_counted = 4 to account for the fact that we need a category that has two courses. Feel free to try something else if there's a better way to go about it